src/__test__/pages/experiments/[experimentId]/data-processing/index.test.jsx

```diff
@@ -126,22 +126,6 @@ const getStore = (settings = {}) => {
 describe('DataProcessingPage', () => {
   const experimentData = {};
 
-  beforeEach(() => {
-    Object.defineProperty(window, 'matchMedia', {
-      writable: true,
-      value: jest.fn().mockImplementation((query) => ({
-        matches: false,
-        media: query,
-        onchange: null,
-        addListener: jest.fn(), // deprecated
-        removeListener: jest.fn(), // deprecated
-        addEventListener: jest.fn(),
-        removeEventListener: jest.fn(),
-        dispatchEvent: jest.fn(),
-      })),
-    });
-  });
-
```

Comment:
Why are these lines removed?

Reply:
`window.matchMedia` is already mocked because we configured jest to set the mock globally. This global mock was already there, but today I am removing the redundant old mocks, and placing the mock file in another folder. See the changes in `jest.config.json`